src/utilities/dockerUtils.ts

```diff
@@ -24,6 +24,9 @@ export async function getDeploymentConfig(): Promise<DeploymentConfig> {
    }
 
    const imageNames = core.getInput('images').split('\n') || []
+   if (imageNames.length == 1 && imageNames[0] == '') {
+      imageNames[0] = '[]'
```

Comment:
setting it to '[]' doesn't make sense here. we'd want the entire array to be empty if the input is all whitespace